examples/inmemory-vectorstore-example/inmemory_vectorstore_example.go

```diff
@@ -0,0 +1,113 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log"
+
+	"github.com/tmc/langchaingo/llms/openai"
+	"github.com/tmc/langchaingo/schema"
+	"github.com/tmc/langchaingo/vectorstores/inmemory"
+
+	"github.com/tmc/langchaingo/embeddings"
+	"github.com/tmc/langchaingo/vectorstores"
+)
+
+func main() {
+	// Create an embeddings client using the OpenAI API. Requires environment variable OPENAI_API_KEY to be set.
+	llm, err := openai.New()
```

Comment:
@corani 
> If you use the cybertron embedder then this would run completely local (modulo having to fetch a model from HF the first time)

This is a great idea. However, I would like to use the openai model to maintain consistency with other vectorstore examples.

Reply:
Sure, that's a valid point :)